### ISSUE TITLE
add pointer to docs for configuration errors

### ISSUE THREAD

**Issue by willkg** [willkg/everett#41]
Right now, if there's a configuration error, you get something like this:

```
ConfigurationError: USERNAME (healthcheck) requires a value parseable by <type 'str'>
```

(The formatting is better, but the gist is the same.)

That's great, but it'd be even more helpful if it included help text or a link or something.

Configuration options for components have help text. We could include that. It might get messy if the help text is large and formatted. Another possibility is to tag a configuration manager with a help url which always gets spit out on configuration errors.

For now, I'm inclined to do the latter. Maybe generalize it so that the configuration manager has a `support_text` field where you could put urls or contact information or whatever.

**Comment by adngdb** [comment]
Something I really like about [require.js](http://www.requirejs.org/) is that every JavaScript error has a link to an explanation about why that error would happen, and ideas to solve the issue. Super, super useful. 

Doing the same with ``everett`` (linking to an RTD page I guess?) would be quite amazing.

**Comment by willkg** [comment]
I don't think we want to point to the Everett RTD page since the target audience for these kinds of configuration errors are people who are trying to set up software that uses Everett for configuration. In the above case, Miles was setting up Antenna on the -dev environment and hit this error. It tells you the specific issue, but doesn't point to the place in the Antenna docs that talk about configuration.

I think adding a `support_text` helps users in this situation: they've configured something incorrectly for some software they're trying to configure which uses Everett and need to be pointed in the right direction--to the software's configuration documentation.

It would also be interesting to include information about that configuration item, too. For components, we have `doc` information about that specific item. I think including that is helpful, but I'm concerned it makes `doc` too hard to write since it has potentially multiple target audiences. Probably worth implementing and then see how it works out and then remove it if it doesn't.
